
From: Brian King <brking@us.ibm.com>

Makes ipr depend on CONFIG_PPC since this driver is unique to PPC hardware.

(It actually builds OK on x86, but it heavily uses anonymous unions, which
breaks on gcc-2.95)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-sparc64-akpm/drivers/scsi/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/scsi/Kconfig~ipr-ppc64-depends drivers/scsi/Kconfig
--- 25-sparc64/drivers/scsi/Kconfig~ipr-ppc64-depends	2004-07-31 20:57:01.316059888 -0700
+++ 25-sparc64-akpm/drivers/scsi/Kconfig	2004-07-31 20:57:01.320059280 -0700
@@ -1010,7 +1010,7 @@ config SCSI_SYM53C8XX_IOMAPPED
 
 config SCSI_IPR
 	tristate "IBM Power Linux RAID adapter support"
-	depends on PCI && SCSI
+	depends on PCI && SCSI && PPC
 	select FW_LOADER
 	---help---
 	  This driver supports the IBM Power Linux family RAID adapters.
_
